Dockerfile

```
@@ -12,6 +12,7 @@ COPY . .src/circuit-knitting-toolbox
 # Fix the permissions of ~/.src
 USER root
 RUN fix-permissions .src
+RUN mkdir persistent-volume && fix-permissions persistent-volume
```

It'd be nice to do this all as one `RUN` statement because each one results in a new layer in the image.
```suggestion
RUN fix-permissions .src && \
    mkdir persistent-volume && fix-permissions persistent-volume
```